packages/prelude/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@most/prelude",
+  "version": "1.5.2",
```

Comment:
I don't think so ... seems simpler not to, but if we have to, that's probably ok.